certbot/tests/util_test.py

```diff
@@ -191,7 +191,10 @@ def test_ok_mode(self):
 
     def test_wrong_mode(self):
         os.chmod(self.tempdir, 0o400)
-        self.assertFalse(self._call(0o600))
+        try:
+            self.assertFalse(self._call(0o600))
+        finally:
+            os.chmod(self.tempdir, 0o700)
```

Comment:
I'm assuming this is for Windows cleanup. Can you add an inline comment explaining that?

Reply:
Done.